Sources/AsyncSequenceValidation/Test.swift

```diff
@@ -412,3 +412,32 @@ extension AsyncSequenceValidationDiagram {
     try self.test(theme: .ascii, build)
   }
 }
+
+final class LockIsolated<Value>: @unchecked Sendable {
```

Comment:
We do have this type already called `ManagedCriticalState`

Reply:
I did not realize that `ManagedCriticalState` was already defined. I have fixed it to use `ManagedCriticalState`.